### DISCUSSION TITLE
Feature Request: Adding a website field in user profiles

### DISCUSSION THREAD

**Discussion by arnavkumr** [bookwyrm-social/bookwyrm#3329] [Ideas]
A specific field where people can add their personal website (or socials) and it will be shown on the user profile. Also, is it possible to make it so that we can [verify](https://joinmastodon.org/verification) our BookWyrm links on our mastodon profile? 

It could work similar to how Pixelfed handles this, we can just add our mastodon profile link to Pixelfed profile's website section and as a result the Pixelfed link becomes verified in the mastodon profile. I don't know how it works in the backend though.

I tried the same with BookWyrm by adding the mastodon profile link in the summary section of the BookWyrm profile but that didn't verify the link on Mastodon profile.

**Comment by dato** [comment]
Hello!

A specific fields sounds a nice addition, yes; in the meantime:

> I tried the same with BookWyrm by adding the mastodon profile link in the summary section of the BookWyrm profile but that didn't verify the link on Mastodon profile.

This ought to work if using the `rel="me"` modifier; like this (I think I've used it in the past, but I could be wrong!):

```html
You can find me at <a rel="me" href="https://masto.example/@myself">@myself@masto.example</a>.
```

**Reply by arnavkumr** [reply]
Oh wow! I didn't know we could use HTML elements like that while writing summary. 

Thank you!
